diagnostic.md

```diff
@@ -32,23 +33,27 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+$("h1").text() = getter and will display the text
+$("h1").text("hello world") = Setter and will set the text
+$("p").html() = getter and will display the <p> element
+$("p").html("hello world") = getter and will set the content of the <p> element
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is essentially like a family tree that can be interacted with. It is the way a document is accessed and manipulated. It allows programmers to modify elements or navigate throughout their "tree."
 <!-- your answer ends here -->
 
 ## Question 6
 
 Please give two examples of things that are commonly confused for the DOM.
 
 <!-- your answer starts here -->
-
+knowing the difference between jQuery setters and getters.
```

Comment:
These are definitely things that people can mess up in jQuery, but we were looking for things that the DOM is confused for, like "html"